### ISSUE TITLE
No Validation Losses?

### ISSUE THREAD

**Issue by DominikVincent** [OpenMotionLab/MotionGPT3#19]
Hello,

Why is it that your code doesn't log validation losses? How do you monitor overfitting? How do you observe convergence?

Did you remove the validation loss logging on purpose, or do you plan to release code that logs validation losses?

**Comment by DominikVincent** [comment]
Thanks for the reply. I see that there are metrics, which in my training seem to be stagnating, e.g. not really any improvement.

<img width="1296" height="573" alt="Image" src="https://github.com/user-attachments/assets/3c6079c9-fa01-45d1-ae8e-4744257ea5b4" />

Hence, I am confused about how you can monitor the training progress of your model, especially if you do not monitor how the loss generalizes to the validation set.

Do you observe the same metric curves? Have you ever verified the validation loss curve?

Thanks for your help :)
